packages/gestalt/src/contexts/DeviceTypeProvider.js

```diff
@@ -3,23 +3,19 @@ import { type Context, type Node, createContext, useContext } from 'react';
 
 const defaultDeviceType = 'desktop';
 
-type DeviceType = 'desktop' | 'phone' | 'tablet';
```

Comment:
nit: I would've saved this change for a follow-up PR just so the preview deploy would work without the reviewer needing to fiddle with anything, but definitely don't revert it now. Just something to keep in mind for the future 🙂